nss-day-cohort-46/holiday-road-accidental-engine
# Kb edit weather

# Description

worked on weatherList to show new weather each time a park is chosen. 
added a parseInt so that temp data comes back as integer without remainder.
hid weather API key.
added my dependency chart to README.md


[ ] New feature (non-breaking change which adds functionality)

# Testing Instructions

git fetch --all
git checkout kb-editWeather
serve
`````````````````````````````
You should see integers only for weather temp and that it refreshes each time a park is chosen. 


# Checklist:

[ ] My code follows the style guidelines of this project
[ ] I have performed a self-review of my own code
[ ] I have commented my code, particularly in hard-to-understand areas
[ ] My changes generate no new warnings or errors
[ ] I have added test instructions that prove my fix is effective or that my feature works
